src/drivers/mct/shr/seq_flds_mod.F90

```diff
@@ -2399,6 +2410,155 @@ subroutine seq_flds_set(nmlfile, ID, infodata)
     call set_glc_elevclass_field(name, attname, longname, stdname, units, x2l_fluxes_from_glc, &
          additional_list = .true.)
 
+    if (trim(cime_model) == 'e3sm') then
```

Comment:
Is this really the appropriate test here or should it be something about active glc model?

Reply:
@jedwards4b , originally @billsacks had wanted much of the OCN-GLC functionality enabled only in E3SM.  In the end we settled on an implementation that avoids needing to know which model is being used and instead functionality is enabled by compset definition via the OCN init routine.  As such, this test could potentially be removed if you and Bill prefer.

Reply:
(I'm sort of back from vacation today, but still taking most of the day off, so may not respond to further discussion until tomorrow.)

As @matthewhoffman says, I requested this change after consultation with @mvertens . We wanted to ensure that these new fields don't impact CESM performance. I suspect that the new flag that @matthewhoffman is referring to isn't available at the time of `seq_flds_set`, so I don't think it will work to use that logical here (but I may be wrong).

Reply:
@billsacks is correct that the needed new flags are not available when `seq_flds_set` is called.  `seq_flds_set` is called from `cime_pre_init2`, which occurs before `cime_init` which is the routine from which the new flags are set (in `ocn_init`).  So I think the current implementation here is the best we can do to prevent these new coupler fields from affecting CESM without significant restructuring.